app/controllers/concerns/submodules_helper.rb

```diff
@@ -0,0 +1,73 @@
+module SubmodulesHelper
+  extend ActiveSupport::Concern
+
+  included do
+    helper_method :active_submodules, :welcome_submodule_active?, :officials_submodule_active?, :agendas_submodule_active?, :blog_submodule_active?, :statements_submodule_active?, :submodule_path_for, :submodule_title_for, :submodule_controller_for
```

Comment:
Now I'm checking this I'm not 100% happy  of not having a scope to GobiertoPeople scope in the methods, but let's see how it looks like when another module needs to be splitted in submodules. It's ok for the time being.